### ISSUE TITLE
No support for Elixir or Erlang languages

### ISSUE THREAD

**Issue by bryanhuntesl** [codota/TabNine#415]
The syntax for Elixir is relatively complex (a certain amount of macro/metaprogramming) but Erlang has a trivial syntax. 
Supporting said languages in 2021 is arguably more useful than support for Perl or Ruby.

**Comment by maya-tabnine** [comment]
Currently we do not support Elixir or Erlang, but you can definitely try using Tabnine with it since it can still be quite helpful in general.
I understand what you are saying, and if we see enough interest, it will be supported.

**Comment by 123mitnik** [comment]
vote for Elixir or Erlang support

**Comment by motionless** [comment]
+1 from my side

**Comment by electr0lulz** [comment]
+1

**Comment by noizu** [comment]
yep, you just lost a few hundred dollars today on this one, although ofcourse it's pennies.
